types/conf.go

```diff
@@ -122,6 +122,13 @@ func LoadDelegateNetConf(bytes []byte, net *NetworkSelectionElement, deviceID st
 		if net.InfinibandGUIDRequest != "" {
 			delegateConf.InfinibandGUIDRequest = net.InfinibandGUIDRequest
 		}
+		if net.DeviceID != "" {
+			if deviceID != "" {
+				logging.Debugf("Warning: Both RuntimeConfig and ResourceMap provide deviceID. Ignoring RuntimeConfig")
```

Comment:
Do you see RuntimeConfig.DeviceID is to replace ResourceMap.DeviceID as default in the future?

Reply:
@zshi-redhat AFAIK, I don't think so. RuntimeConfig.DeviceID is useful for cases in which you don't have Device Plugin (hence, no ResourceMap.DeviceID)